### DISCUSSION TITLE
Greek symbols group titles on symbols file

### DISCUSSION THREAD

**Discussion by rgbatista** [joaomlourenco/novathesis#509] [General]
Dear someone who might be able to help,

I would like to know if it is possible to have group titles like for example for greek symbols, which in my case is for greek symbols. Below is an example of I'm trying to use.

![image](https://github.com/user-attachments/assets/fa8b89e9-7b22-4acd-9acc-ae99a9a012c7)

![symbols](https://github.com/user-attachments/assets/38ae337a-d455-49cd-9e29-bf5d0be41c10)

Has anyone ever encountered or heard of a similar problem? Or does know a solution to this problem?

If someone could offer insights or assistance, I would be extremely grateful!

Best Regards,
Rafael Batista

**Comment by joaomlourenco** [comment]
I am not sure what you want. :( 
Do you have a glossary named "Nomenclature" already?  Is it a new one or you just "renamed" one of the standard ones (e.g.., you renamed the "symbols" list)?
Do you have multiple sub-titles? Or just one? (greek letters)

Have a look at this post and see if it can help you can devise a solution from your problem!

https://tex.stackexchange.com/questions/99142/customising-glossary-group-titles
